src/ast/mod.rs

```diff
@@ -670,6 +670,12 @@ pub enum Statement {
         columns: Vec<Ident>,
         /// VALUES a vector of values to be copied
         values: Vec<Option<String>>,
+        /// file name of the data to be copied from
+        filename: Option<Ident>,
+        /// delimiter character
+        delimiter: Option<Ident>,
+        /// CSV HEADER
```

Comment:
I didn't think `CSV HEADER` was part of the syntax. Postgres seems to only support `HEADER`:

https://www.postgresql.org/docs/13/sql-copy.html

I think CSV is used as part of the FORMAT option, like

```sql
COPY ... FORMAT CSV ...
```

Reply:
![compactability](https://user-images.githubusercontent.com/10010900/152666949-e4451b2f-e2c5-4703-8359-dc38f72b39fa.png)

If you scroll down, you'll find the compatibility section. There you can find CSV HEADER spec. 

Sure, I merge the main